source/Playnite/Settings/DetailsVisibilitySettings.cs

```diff
@@ -83,6 +83,21 @@ public bool LastPlayed
             }
         }
 
+        private bool added = true;

```

Comment:
I don't think this should be enabled by default. It's obviously not that important since nobody noticed until now that we don't even have an option for this :)

Reply:
Personally I would have preferred to make it enabled so people know of its existence but changed in https://github.com/JosefNemec/Playnite/pull/3316/commits/5ba93435980107d5ddeefb8b4db307c415bc1a62